skimage/transform/_geometric.py

```diff
@@ -463,6 +463,16 @@ class EssentialMatrixTransform(FundamentalMatrixTransform):
     params : (3, 3) array
         Essential matrix.
 
+    Examples
+    --------
+    >>> tform_matrix = EssentialMatrixTransform(
```

Comment:
@lagru I'm surprised that this works (there is no CI failure), because the [Examples docs](https://numpydoc.readthedocs.io/en/latest/format.html#examples) say: "All other imports, including the demonstrated function, must be explicit."

I remember that this question came up 'recently' (when we realized that doctests weren't actually running), but I don't remember what the final word was!

Reply:
I don't think there was a final word (also don't remember were it came up). But my 2cts is to make these examples as self-contained as possible by importing the tested function as well. Meaning that you can just copy them into an empty Python console and run.

Reply:
Hmm, should I add the imports to the examples as well?

Reply:
It wouldn't hurt. :)